server/routes/users.js

```diff
@@ -63,7 +64,7 @@ router.route("/register").post((req, res) => {
   //ENCRYPT PASSWORD
   password = bcrypt.hashSync(req.body.password, 10);
   //CREATE NEW USER
-  const newUser = new User({ _id, name, email, password });
+  const newUser = new User({ _id, name, email, password, isChef });
```

Comment:
Can't recall if we talked about it but are we just providing a way for someone to become a chef after they had already signed up? From this I see every signup will set isChef to false, in which case it may be cleaner to set the default value for the field to be false in server/models/user.js

Reply:
I don't think we've talked about becoming a chef on this site. So if the option to become a chef is after sign up, I agree that it would be cleaner for a default value to be set in the models.